src/WarcraftLegacies.Source/Setup/FactionSetup/LordaeronSetup.cs

```diff
@@ -98,10 +98,10 @@ public static void Setup(PreplacedUnitSystem preplacedUnitSystem)
 
       //Todo: these probably should be in some kind of ability library, not here
       Lordaeron.ModAbilityAvailability(Constants.ABILITY_A0N2_GRASPING_VINES_TREANTS, -1);
-      Lordaeron.ModAbilityAvailability(Constants.ABILITY_A0GD_SUMMON_GARRISON_STORMWIND, -1);
```

Comment:
What's the removal for? Is the intent that Lordaeron should be able to use Stormwind's Summon Garrison if they get control of one of their buildings?

Reply:
i brain freezed